stetho/src/main/java/com/facebook/stetho/inspector/database/DatabasePeerManager.java

```diff
@@ -30,18 +30,37 @@
   };
 
   private final Context mContext;
+  private final DatabaseFilesProvider mDatabaseFilesProvider;
 
-  public DatabasePeerManager(Context context) {
+  /**
+   * Constructs the object with a {@link DatabaseFilesProvider} that supplies the database files
+   * from {@link Context#databaseList()}.
+   *
+   * @param context the context
+   * @deprecated use
+   */
+  @Deprecated
+  public DatabasePeerManager(final Context context) {
+    this(context, new DefaultDatabaseFilesProvider(context));
+  }
+
+  /**
+   * @param context the context
+   * @param databaseFilesProvider a database file name provider, can be null
+   */
+  public DatabasePeerManager(Context context, DatabaseFilesProvider databaseFilesProvider) {
```

Comment:
Also, mark `mDatabaseFilesProvider` with `@Nullable` and then make things not crash if it's null.  Or drop this null support :)